.gitignore

```diff
@@ -0,0 +1,33 @@
+.DS_Store
```

Comment:
Quite a few of these elements are not needed. Perhaps start with a blank file, or one with just for `build` dir and add from there.